index.html

```diff
@@ -15,55 +15,49 @@
 			<input type="search" class="text-field" />
 			<div class="navbar-cta">
 				<a id="login-text" class="btn btn-link">Hello, Login</a>
-				<div class="badge-holder">
-					<svg
-						xmlns="http://www.w3.org/2000/svg"
-						xmlns:xlink="http://www.w3.org/1999/xlink"
-						aria-hidden="true"
-						role="img"
-						class="iconify iconify--ph"
-						width="32"
-						height="32"
-						preserveAspectRatio="xMidYMid meet"
-						viewBox="0 0 256 256"
-					>
-						<path
-							d="M221.7 72l-26.4 92.4a15.9 15.9 0 0 1-15.4 11.6H84.1a15.9 15.9 0 0 1-15.4-11.6L42.3 72z"
-							opacity=".2"
-							fill="currentColor"
-						></path>
-						<path
-							d="M96 216a16 16 0 1 1-16-16a16 16 0 0 1 16 16zm88-16a16 16 0 1 0 16 16a16 16 0 0 0-16-16zm45.4-125.8L203 166.6a24.1 24.1 0 0 1-23.1 17.4H84.1A24.1 24.1 0 0 1 61 166.6L34.6 74.3v-.2L24.8 40H8a8 8 0 0 1 0-16h16.8a16.1 16.1 0 0 1 15.4 11.6L48.3 64h173.4a8 8 0 0 1 7.7 10.2zM211.1 80H52.9l23.5 82.2a8 8 0 0 0 7.7 5.8h95.8a8 8 0 0 0 7.7-5.8z"
-							fill="currentColor"
-						></path>
-					</svg>
-					<div class="badge-icon">1</div>
-				</div>
+				<svg
```

Comment:
just wanted to ask you, why inline SVG for icons?
svgs can be separated into separate files and can be used with img tag
line of codes will be shorter

Reply:
Adding SVG inside img tag won't change the colour on the hover. That's why.

Reply:
yup zoef same question